snippets/csharp/VS_Snippets_CLR_System/system.io.enumdirs1/cs/program.cs

```diff
@@ -10,9 +10,10 @@ private static void Main(string[] args)
     {
         try
         {
-            string dirPath = @"\\archives\2009\reports";
+            // Set a variable to the My Documents path.
+            string mydocpath = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
same comment about camel-casing and possibly shortening variable name to docPath